src/cf.data.pre

```diff
@@ -1043,6 +1043,11 @@ DOC_START
 	  # NOTE 2: IPv6 protocol does not contain ARP. MAC/EUI is either
 	  # encoded directly in the IPv6 address or not available.
 
+	acl aclname client_connmark mark[/mask] ...
+	  # [fast]
+	  # check client's connection mark
```

Comment:
There needs to be a mention here that connection markings are only valid on Linux when Squid is built --enable-linux-netfilter and netfilter-conntrack library is used.

Also the Netfilter feature is called CONNMARK upper case.